app/src/main/kotlin/com/wire/android/ui/home/appLock/forgot/ForgotLockCodeResetDeviceDialog.kt

```diff
@@ -51,6 +52,7 @@ import com.wire.android.util.ui.stringWithStyledArgs
 @Composable
 fun ForgotLockCodeResetDeviceDialog(
     username: String,
+    isPasswordNotRequired: Boolean,
```

Comment:
`suggestion`: For a better reading wouldn't it be better if this param was positive? as `isPasswordRequired` ?

Reply:
sure, you are right, thanks 👍